src/main/java/org/spring/FullVersion/Main.java

```diff
@@ -1,38 +1,171 @@
 package org.spring.FullVersion;
 
-
+import java.time.LocalDate;
+import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) {
-
-        FamilyDao familyDao = new CollectionFamilyDao();
-        FamilyService familyService = new FamilyService(familyDao);
-        FamilyController familyController = new FamilyController(familyService);
+    public static void main(String[] args) throws FamilyOverflowException {
 
         Human father = new Human("John", "Doe", 1980);
         Human mother = new Human("Jane", "Doe", 1982);
-        familyController.createNewFamily(father, mother);
-
-        Human father2 = new Human("John", "Doe", 1981);
-        Human mother2 = new Human("Jane", "Doe", 1983);
-        familyController.createNewFamily(father2, mother2);
-
-
-        familyController.displayAllFamilies();
-
-        familyController.getFamiliesBiggerThan(0);
 
+        Scanner scanner = new Scanner(System.in);
+        FamilyService familyService = new FamilyService(new CollectionFamilyDao());
+
+        while (true) {
+            displayMenu();
+            System.out.print("Enter your choice: ");
+            String command = scanner.nextLine();
+
+            switch (command) {
+                case "1":
+                    fillWithTestData(familyService);
+                    break;
+                case "2":
+                    familyService.displayAllFamilies();
+                    break;
+                case "3":
+                    System.out.println("Enter the number of people: ");
+                    int greaterThan = Integer.parseInt(scanner.nextLine());
```

Comment:
you can write one common getInput method, (including all validations) and then use this method when needed